core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -685,7 +685,7 @@ public synchronized void startInternal() throws InterruptedException, IOExceptio
       mServer.start();
     } catch (IOException e) {
       String errorMessage = ExceptionMessage.FAILED_RAFT_BOOTSTRAP
-          .getMessage(Arrays.toString(clusterAddresses.toArray()), e.getCause().toString());
+          .getMessage(Arrays.toString(clusterAddresses.toArray()), e);
```

Comment:
I'm not sure what the original intent was, but the new code changes the original behavior. The original would print the cause, but this PR will print the original exception.

Maybe we print the `e.getCause()` if it is not `null`, and `e` otherwise?

Reply:
Sounds good.

Reply:
> I'm not sure what the original intent was, but the new code changes the original behavior. The original would print the cause, but this PR will print the original exception.
> 
> Maybe we print the `e.getCause()` if it is not `null`, and `e` otherwise?

Right! It is the right way.It can give users more useful info about the exception rather than null pointer exception.